solidity/test/rewards/TestECDSARewards.js

```diff
@@ -115,20 +115,22 @@ describe("ECDSARewards", () => {
     it("should not be possible when a keep is not closed", async () => {
       await keepFactory.stubOpenKeep(owner, operators, firstIntervalStart)
       const keepAddress = await keepFactory.getKeepAtIndex(0)
+      const keepAddresses = [keepAddress]
 
       const isEligible = await rewardsContract.eligibleForReward(keepAddress)
       expect(isEligible).to.be.false
 
       await timeJumpToEndOfInterval(0)
       await expectRevert(
-        rewardsContract.receiveReward(keepAddress),
+        rewardsContract.receiveReward(keepAddresses),
```

Comment:
Let's rename the new function to `receiveRewards` (plural) and we'll be able to avoid overloading problems.

Reply:
Done